flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
Why restrict to x86_64? You are building from source?

Reply:
Because the games currently only run on x86-64 which makes the launcher pointless for any other architecture

Reply:
Ok